TF_Tooling_Web/src/tests/TouchFreeToolingTypes.test.ts

```diff
@@ -65,6 +65,10 @@ const bitmaskFlagParams: BitmaskFlags[] = [
 ];
 
 describe('BitmaskFlag', () => {
+    // Suppress errors from console and store them in an array which we print only if a test fails
+    const errors: string[] = [];
+    console.error = jest.fn((msg: string) => errors.push(msg));
```

Comment:
Do we need to revert this so that we get other errors from other tests or is the environment reset on exiting the `describe`?

Reply:
Fixed - the environment is set per test file it seems